eclipse_plugin/pom.xml

```diff
@@ -0,0 +1,96 @@
+<!--
+  ~ Copyright 2015 Google Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.googlejavaformat</groupId>
+    <artifactId>google-java-format-parent</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>google-java-format-eclipse-plugin</artifactId>
+  <version>0.0.1</version>
+  <packaging>eclipse-plugin</packaging>
+  <name>Google Java Format Plugin for Eclipse 4.5+</name>
+
+  <description>
+    A Java source code formatter that follows Google Java Style.
+  </description>
+
+  <properties>
+    <!-- Tycho 0.25 requires Java 8 to run -->
```

Comment:
Upgraded to 0.26, worked locally. Let's see what Travis and AppVeyor will do with it.